lib.sh

```diff
@@ -683,13 +683,30 @@ calc_wt_size() {
     export WT_MENU_HEIGHT
 }
 
+# example: is_app_enabled documentserver_community
+is_app_enabled() {
+install_if_not jq
+if occ_command app:list --output=json | jq -e ".enabled | .$1" > /dev/null
+then
+    return 0
+else
+    return 1
+fi
+}
+
 install_and_enable_app() {
+# Check if app is already installed
+if is_app_enabled "$1"
+then
+    return
```

Comment:
What does this addition do?

Reply:
More importantly, why is it necessary?

Reply:
I think, that we don't have to inform the user if the app is enabled: then we just skip the whole install process and don't try to enable the app since it is already installed.

Reply:
No, please leave it is as.

Reply:
this is reverted